pypeman/tests/test_nodes.py

```diff
@@ -636,6 +669,13 @@ def test_csv2python(self):
         self.assertListEqual(self.py_data, processed_msg.payload)
         return msg
 
+    def test_csvstr2python(self):
+        csv2py_node = nodes.CSVstr2Python(to_dict=True, headers=True)
```

Comment:
make also a test where headers is False and one where to_dict is false

Reply:
done